File: PicassoKutuphanesi/app/src/main/java/com/example/picassokutuphanesi/MainActivity.java
package com.example.picassokutuphanesi;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class MainActivity extends AppCompatActivity {
    ImageView img;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tanimla();
        resimCek();
    }
    public void tanimla(){
        img=(ImageView) findViewById(R.id.img);
    }
    public void resimCek(){
        Picasso.with(getApplicationContext()).load("https://i.hizliresim.com/xYggpk.png").into(img);
    }
}

File: KotlinDers17AddMobAndroid/app/src/main/java/com/example/kotlinders17addmobandroid/MainActivity.kt
package com.example.kotlinders17addmobandroid

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.google.android.gms.ads.AdRequest
import com.google.android.gms.ads.AdView
import com.google.android.gms.ads.MobileAds

/**
 * app id: ca-app-pub-4718213835461634~9920500742
 * reklam id: ca-app-pub-4718213835461634/7287115467
 * banner test id: 	ca-app-pub-3940256099942544/6300978111

 */
class MainActivity : AppCompatActivity() {

    lateinit var mAdView : AdView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        MobileAds.initialize(this) {}
        mAdView = findViewById(R.id.adView)
        val adRequest = AdRequest.Builder().build()
        mAdView.loadAd(adRequest)
    }
}
